x-pack/plugins/apm/server/lib/observability_overview/get_transaction_coordinates.ts

```diff
@@ -63,12 +64,10 @@ export async function getTransactionCoordinates({
     },
   });
 
-  const deltaAsMinutes = (end - start) / 1000 / 60;
-
   return (
     aggregations?.distribution.buckets.map((bucket) => ({
       x: bucket.key,
-      y: bucket.count.value / deltaAsMinutes,
+      y: getTpmRate(setup, bucket.count.value),
```

Comment:
nit: I think it would be slightly better like this: 

```suggestion
      y: getTpmRate({ start, end, value: bucket.count.value }),
```

(it's most passing `setup` as an argument that feels off)